### ISSUE TITLE
Add variable assert statement into all roles

### ISSUE THREAD

**Issue by mbukatov** [Tendrl/tendrl-ansible#91]
We may consider adding [assert task checking ansible variables](https://thoughts.t37.net/documenting-your-ansible-roles-interface-and-making-other-peoples-life-easier-604e5172837b) like this:

```
init.yml

- assert:
    that:
      - db_host != ''
      - db_port != ''
      - db_user != ''
      - db_user != 'root'
      - db_password != ''
      - db_name != ''
  tags: ['check_vars']
```

It would provide:

* clear error message about mandatory variables
* way to check variables before running the playbook: `ansible-playbook -i hosts -t check_vars site.yml`

**Comment by mbukatov** [comment]
We have something similar, but in site.yml playbook file:

```
    - name: Check that mandatory variables are defined
      assert:
        that:
          - etcd_fqdn is defined
          - etcd_ip_address is defined
          - graphite_fqdn is defined
```

**Comment by mbukatov** [comment]
Implemented in f648062eb3a4d6ed22bddc9e0c064f24bc0972e8
